kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -470,9 +503,21 @@ define([
             requestLatestJobLog();
         }
 
+        /**
+         * toggle the viewer class to switch between standard and expanded versions
+         */
         function toggleViewerSize() {
-            panelHeight = panelHeight === smallPanelHeight ? largePanelHeight : smallPanelHeight;
-            getLogPanel().style.height = panelHeight;
+            let logContentClassList = getLogPanel().classList,
```

Comment:
This switches classes instead of switching out panel heights. Looking at how much longer the code is now, I wonder why I bothered with this...